core-bootstrap/src/components/accordion/accordion.ts

```diff
@@ -114,11 +114,11 @@ const accordionItemDefaultConfig: AccordionItemProps = {
 const accordionItemProps = Object.keys(accordionItemDefaultConfig);
 
 /**
- * Create an AccordioItemnWidget with given config props
- * @param config - an optional alert config
+ * Create an AccordionItemWidget with given config props
+ * @param config - an optional AccordionItem config
  * @returns an AccordionWidget
  */
-export const createAccordionItem = extendWidgetProps(createCoreAccordionItem, defaultConfigItemExtraProps);
+export const createAccordionItem: WidgetFactory<AccordionItemWidget> = extendWidgetProps(createCoreAccordionItem, defaultConfigItemExtraProps);
```

Comment:
I notice that we (well, **I**) forgot the jsdoc for the `createAccordion` in core-bootstrap.
Do you mind adding in this PR ?